eng/pipelines/templates/steps/build-test.yml

```diff
@@ -20,9 +20,15 @@ steps:
     env:
       GO111MODULE: 'on'
 
-  - pwsh: ./eng/scripts/run_tests.ps1
-    displayName: 'Run Tests and Create Coverage'
-    workingDirectory: '${{parameters.GoWorkspace}}'
+  - task: PowerShell@2
+    inputs:
+      targetType: 'filePath'
+      filePath: '/eng/scripts/run_tests.ps1'
+      workingDirectory: '${{parameters.GoWorkspace}}'
+      arguments: '${{parameters.ServiceDirectory}}'
```

Comment:
If you switch to the task directly you will need to set `pwsh: true` to run on powershell core if this runs on anything other than windows.